packages/corvid-cli/test/clone.e2e.js

```diff
@@ -401,4 +402,34 @@ describe("clone", () => {
       );
     });
   });
+  describe("Failed clone", () => {
+    it("should clean files", async () => {
```

Comment:
we need another tests that makes sure we don't delete other files that were in the root folder and not created by the clone command

Reply:
done

Reply:
done